# Part 1 Answers

1. "values added: 20" is printed by line 9. 
2. "final result: 20" is printed by line 13.
3. "values added: 20" is printed by line 9.
4. An Uncaught ReferenceError occurs on line 13. This is because result is not defined, since it was declared using "let", and line 13 is outside the block in which it was declared
5. Nothing is printed on line 9, there is an Uncaught TypeError on line 7. This is because the program tries to assign a new value to a variable that was declared with the const keyword.
6. Nothing is printed on line 13 because there was the error earlier in the program and the program terminated.